projects/plugins/boost/app/assets/src/js/utils/analytics.ts

```diff
@@ -73,3 +73,12 @@ function addBoostProps( props: TracksEventProperties ): TracksEventProperties {
 
 	return { ...defaultProps, ...props };
 }
+
+export async function recordBoostEventAndRedirect(
+	url: string,
+	eventName: string,
+	eventProp: TracksEventProperties
```

Comment:
I noticed that a lot of places where the function is called, it's with an empty object. Maybe that should be the default value of `eventProp`?

Reply:
It called recordBoostEvent() immediately where the second parameter is a TracksEventProperties object. I'd rather leave it that way.

Reply:
I won't be proscriptive over it, but I agree with @dilirity - it seems that we have a lot of events with no props, so setting a default value of `{}` makes sense to me.

Reply:
Done!